addons/website_sale_slides/views/website_sale_templates.xml

```diff
@@ -3,13 +3,14 @@
 
 <template id="website_sale_confirmation_slide" inherit_id="website_sale.confirmation">
     <xpath expr="//div[@id='oe_structure_website_sale_confirmation_2']" position="after">
-        <t t-call="website_sale_slides.course_purchased_confirmation_message"/>
+        <t t-if="order.order_line.product_id.channel_ids" t-call="website_sale_slides.course_purchased_confirmation_message"/>
```

Comment:
Shouldn't we use the ``detailed_type`` field (being ``course``) ? That way we avoid looking up into a m2m.